tests/source/test_csv.py

```diff
@@ -9,3 +9,10 @@
 class TestCSVSource(SourceTest, AsyncTestCase):
     async def setUpSource(self, fileobj):
         return CSVSource(FileSourceConfig(filename=fileobj.name))
+
+class CSVTest(SourceTest, AsyncTestCase):
+    async def setUpSource(self, fileobj):
+        return CSVSource(FileSourceConfig(filename=fileobj.name, key="UniqueFeature"))
+
+    async def test_key(self):
+        file =
```

Comment:
Now make a csv file, you could just do

```python
with tempfile.NamedTemporaryFile() as fileobj:
    fileobj.write(b"KeyHeader,ValueColumn\n")
    fileobj.write(b"a,42\n")
    fileobj.write(b"b,420\n")
    fileobj.seek(0)
    async with CSVSource(FileSourceConfig(filename=fileobj.name, key="KeyHeader")) as source:
        async with source() as sctx:
        # Check that the values for ValueColumn are correct for each repo (a, b)
```

Reply:
Oh okay. I've done in a different way. I didnt see this. I'll change it now.

Reply:
Done :+1: